src/components/common/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+const Button = ({text,onClick, disabled=false}) => {
```

Comment:
I don't see much value in this component, I think you can remove it and move the code directly where it's used.